app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -472,12 +477,19 @@ export default {
     saveDraft(conversationId, replyType) {
       if (this.message) {
         const savedDraftMessages = this.getSavedDraftMessages();
-        LocalStorage.set(LOCAL_STORAGE_KEYS.DRAFT_MESSAGES, {
-          ...savedDraftMessages,
-          [`draft-${conversationId}-${replyType}`]: trimMessage(
-            this.message || ''
-          ),
-        });
+        const key = `draft-${conversationId}-${replyType}`;
+        const draftToSave = trimMessage(this.message || '');
+
+        const currentDraft = savedDraftMessages[key];
+
+        if (currentDraft !== draftToSave) {
```

Comment:
I don't think this check is required. We can overwrite the content without a check.